### ISSUE TITLE
Most recent commit unresponsive randomy

### ISSUE THREAD

**Issue by Emlorp** [cursey/kanan-new#13]
I saw you added a few issues and one was persistent. Client devcat when active prevents harvesting on my human without using harvest song (endless animation) fixed by disabling devcat temporarily. The other seems to be the menu just at times is completely unresponsive and/or cannot be clicked displayed or not. It will instead interact with anything behind the menu

**Comment by cursey** [comment]
The issue with the Client Side Devcat patch is just something you have to live with if you want to use that mod (I am not the author of the mod) which is why it comes with a warning. 

I can't replicate the issue you're having with the unresponsive UI. My guess is something (like the steam overlay or similar) is causing this as a side effect of how kanan hooks user input.  Do you run any other overlay type programs?

**Comment by cursey** [comment]
Actually you say this just started happening with the most recent commit? I did change how input is captured based on the UI wanting input. That probably has something to do with it. I'll look into it more.

**Comment by Emlorp** [comment]
Yeah the former I accept the issue with and can live with i must have missed the warning. As for the one with this commit if it doesnt replicate on your end I could record the issue from a launch to in game.. No overlays currently used aside from Kanan.
